### ISSUE TITLE
[Bug] PowerShellImport / PowerShell not behaving

### ISSUE THREAD

**Issue by rasta-mouse** [cobbr/Covenant#72]
**Feature Request or Bug**
Bug

**Describe the feature request or bug**
The PowerShell function imported via PowerShellImport does not appear to execute when calling it via the PowerShell command.

**To Reproduce**

Test script:
```
function Test-Cmdlet {
    New-Item -Path C:\tmp\ -Name test.txt -ItemType "file" -Force
}
```

1. PowerShellImport
2. Select `Test-Cmdlet.ps1` from file browser
3. PowerShell Test-Cmdlet

**Expected behavior**
`test.txt` should be written to `C:\tmp\`.  The directory does exist on the target and running the test script manually on the host produces expected results.

**Screenshots**
![image](https://user-images.githubusercontent.com/7346521/65540113-aa346580-df02-11e9-9fe0-205590b894cd.png)

**Covenant Server Information:**
 - OS: Kali Linux Rolling 2019.4
 - Docker or Native: Native

**Browser Information:**
 - Google Chrome
 - Version 77.0.3865.90 (Official Build) (64-bit)

**Target Information (System that implant is running on):**
 - OS: Windows 10
 - Version 1809 (Build 17763.737)

**Additional context**
I'm sorry that I suck.

**Comment by cobbr** [comment]
You don't suck 😛 

I have not tested yet, but I have a suspicion about what could be causing this. Try doing PowerShellImport with a filename. For instance, `PowerShellImport test.ps1`, then upload via file dialog. Filename shouldn't matter.

**Comment by rasta-mouse** [comment]
Same result :(

I did notice a Unicode character in the `PowerShellCommand` box in the GruntTasking window.  Not sure if that's also being passed through to the actual code that's been executed?  I can enable PowerShell logging on the host and check.

![image](https://user-images.githubusercontent.com/7346521/65541725-c84f9500-df05-11e9-8bc3-c4d56ab1f7d0.png)

**Comment by cobbr** [comment]
Hmm yeah that's worth checking

**Comment by rasta-mouse** [comment]
I'm going to go with yes.  Even though you can't see the character in the transcript logs, this is what you get.
```
**********************
PS>CommandInvocation(Out-String): "Out-String"
>> ParameterBinding(Out-String): name="InputObject"; value="The term '﻿function' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again."
﻿function : The term '﻿function' is not recognized as the name of a cmdlet, function, script file, or operable program. 
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ ﻿function Test-Cmdlet {
+ ~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (﻿function:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
﻿function : The term '﻿function' is not recognized as the name of a cmdlet, function, script file, or operable program. 
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ ﻿function Test-Cmdlet {
+ ~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (﻿function:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
**********************
```

I created an identical test script on Linux and that one worked.

![image](https://user-images.githubusercontent.com/7346521/65544508-45313d80-df0b-11e9-8cda-d1898b22db54.png)

Any PowerShell script I create in the PowerShell ISE on Windows seems to consistently have this Unicode character at the beginning of the file.  Based on my preliminary reading, this may be some Byte Order Mark magic number.

**Comment by cobbr** [comment]
Strange, I suppose I can handle trimming off that specific character.

**Comment by rasta-mouse** [comment]
I'm surprised nobody else has run into this - it can't be that unusual to use the ISE for Intellisense etc?  I'd be interested to see if you or anyone else can reproduce it consistently.

**Comment by cobbr** [comment]
Very strange. When I use your exact cmdlet, I get the same issue.

When I use another cmdlet, also created with the ISE, all is well:
![Screen Shot 2019-09-28 at 3 49 00 PM](https://user-images.githubusercontent.com/8904306/65822321-a2790780-e207-11e9-8aa7-4140db485837.png)

**Comment by cobbr** [comment]
I take it back. It's interpreting the entire line as a string for some reason lol

**Comment by cobbr** [comment]
Should be fixed in 8a1e68079c0b4465ae43a720c9fe2a1753c25610

Thanks @rasta-mouse!
